src/main/java/com/coveros/selenified/OutputFile.java

```diff
@@ -374,9 +374,9 @@ public void recordAction(String action, String expectedResult, String actualResu
             out.write("    <td align='center'>" + stepNum + ".</td>\n");
             out.write(START_CELL + action + END_CELL);
             out.write(START_CELL + expectedResult + END_CELL);
-            out.write("    <td class='" + result.toString().toLowerCase() + "'>" + actualResult + imageLink + END_CELL);
+            out.write("    <td>" + actualResult + imageLink + END_CELL);
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Use already-defined constant 'START_CELL' instead of duplicating its value here. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS1192)